feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_ovsdb.c

```diff
@@ -415,6 +423,11 @@ void rrm_update_rrm_config_cb(ovsdb_update_monitor_t *self)
 		rrm_config->schema.snr_percentage_drop = 0;
 		rrm_config->schema.client_disconnect_threshold = 0;
 		rrm_config->schema.probe_resp_threshold = 0;
+		rrm_config->schema.noise_floor_thresh = 0;
+		rrm_config->schema.noise_floor_time = 0;
+		rrm_config->schema.non_wifi_thresh = 0;
+		rrm_config->schema.non_wifi_time = 0;
+		rrm_config->schema.obss_hop_mode = 0;
 
```

Comment:
Should we collapse this whole thong to a memset 0 on the structure?

Reply:
Looks to be safe to do a memset 0. We are anyways removing/deleting/freeing the whole of rrm_config structure.